debug/examples/shop-demo.json5

```diff
@@ -50,6 +50,11 @@
         x: 18.86,
         y: 15.09,
     },
+    menu: {
+        buttonsRight: true, // set as false to X-flip the menu layout
+        buttonBackgroundKey: "menuButtonBackground",
+        buttonIconKey: "menuButtonIcon",
```

Comment:
prob could get away with backgroundKey and iconKey

Reply:
I moved these and simplified the names: ff0de05